index.html

```diff
@@ -116,17 +115,18 @@ <h2>Main Program<hr></h2>
                     </div> 
                 </div>
                 <p class="main-down-word">SEE THE WHOLE PROGRAM</p>
+                <button class="joincc">Join CC Global Summit 2015</button>
     </section>
     </section>
     <section class="featured-speakers-heading">
             <div class="featured-speakers-heading">
-                <h5>Featured Speakers</h5>
-                <div id="line"></div>
+                <h5>Featured Speakers<hr></h5>
             </div>
             <div id="featured-speakers"></div>
             <div class="more-mobile">
-                <p>MORE</p>
-                <img src="imgs/more-arrow.png">
+            <button>
+                <p>MORE<img src="icon/arrow_down 1.png"></p>
+            </button>
```

Comment:
- [ ] It appears that there is no functionality attached to this yet and the full list of speakers is showing, HOw about just removing it completely or adding the functionality to truncate or slice the list until clicked?

Reply:
Yeah, I'll just remove it per now and add it later please!